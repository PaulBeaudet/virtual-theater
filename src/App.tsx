import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import './App.scss';
import Auth from './components/Auth';
import Theater from './components/Theater';
import { GlobalUserProvider } from './context/GlobalState'

const App = () => {
  return (
    <GlobalUserProvider>
      <Router>
        <Switch>
          <Route exact={true} path='/' component={Auth} />
          <Route exact={true} path='/theater' component={Theater} />
        </Switch>
      </Router>
    </GlobalUserProvider>
  );
};

export default App;
