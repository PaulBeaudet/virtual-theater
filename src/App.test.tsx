import React from 'react';
import ReactDOM from 'react-dom'
// import { render } from '@testing-library/react';
import renderer from 'react-test-renderer'
import App from './App';

test('app snapshot test', () => {
  const component = renderer.create(<App />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<App />, div)
})
