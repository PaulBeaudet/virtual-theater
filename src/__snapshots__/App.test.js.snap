// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke test 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "height": "100vh",
      "justifyContent": "center",
    }
  }
>
  <h1>
     Remo Coding Challenge Join Room 
  </h1>
  <button
    onClick={[Function]}
  >
     Login With Google 
  </button>
</div>
`;
